src/components/Gradebook/index.jsx

```diff
@@ -59,7 +59,11 @@ export default class Gradebook extends React.Component {
           earned_graded_override: this.state.updateVal,
         },
       },
-    ]);
+    ],
+    this.state.filterValue,
```

Comment:
The indentation here doesn't look right. Please correct.

Reply:
I'm not sure what you mean. What should I change here?

Reply:
So, the ] should be combined with the } above. Then you can have the values indented a "tab" in.
Anyways. Not a big deal.